### PR TITLE
New version: ente-io.ente version 2.0.50

### LINKED ISSUE #147677: [Package Request]: ente-auth
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/ente-io/ente
Publisher:ente-io
Package Name: ente-auth
Description: open source end-to-end encrypted authenticator app
Package Version: v2.0.50
Installer URL: https://github.com/ente-io/ente/releases/download/auth-v2.0.50/ente-auth-v2.0.50-installer.exe

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.0 :rocket:

- Resolve #147677
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/147679)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\auth.exe returned exit code: -1073741515

(Automated response - build 876.)

File: manifests/e/ente-io/ente/2.0.50/ente-io.ente.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-03-29
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/ente-io/ente/releases/download/auth-v2.0.50/ente-auth-v2.0.50-installer.exe
+  InstallerSha256: EBF3F660D241C8D9363A4C0FB2518312DC0501D1870AF532F7204A7C368F5D07
```
Comment (stephengillie) [inline manifests/e/ente-io/ente/2.0.50/ente-io.ente.installer.yaml:16]:
```suggestion
  InstallerSha256: EBF3F660D241C8D9363A4C0FB2518312DC0501D1870AF532F7204A7C368F5D07
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/e/ente-io/ente/2.0.50/ente-io.ente.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: ente-auth-v2.0.50-windows/auth.exe
+  InstallerUrl: https://github.com/ente-io/ente/releases/download/auth-v2.0.50/ente-auth-v2.0.50-windows.zip
+  InstallerSha256: EF39FDFAA201A1039B66F0F267204AC0F4946C2E028014AFDAF07845B6177F2A
```
Comment (stephengillie) [inline manifests/e/ente-io/ente/2.0.50/ente-io.ente.installer.yaml:23]:
```suggestion
  InstallerSha256: EF39FDFAA201A1039B66F0F267204AC0F4946C2E028014AFDAF07845B6177F2A
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Exorcism0666,

This software package appears to depend on `Microsoft.VCRedist.2015+.x86` / `x64`: 
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
(Automated response - build 612.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-07 23:38:06.492 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!69E3535B: (caller: 69D31A80) LogHr(1) tid(628) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!69E7B5DD: (caller: 69F112BF) Exception(1) tid(628) 8A15005E ] 
> 2024-04-07 23:38:06.492 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:13.899 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!55F8535B: (caller: 55E81A80) LogHr(1) tid(504) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!55FCB5DD: (caller: 560612BF) Exception(1) tid(504) 8A15005E ] 
> 2024-04-07 23:38:13.899 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:12.428 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!69DB535B: (caller: 69CB1A80) LogHr(1) tid(1410) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!69DFB5DD: (caller: 69E912BF) Exception(1) tid(1410) 8A15005E ] 
> 2024-04-07 23:38:12.429 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:19.626 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!55B0535B: (caller: 55A01A80) LogHr(1) tid(1060) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!55B4B5DD: (caller: 55BE12BF) Exception(1) tid(1060) 8A15005E ] 
> 2024-04-07 23:38:19.627 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:20.863 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x86 with version 14.38.33135.0

(Automated response - build 876.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-04-07 23:38:06.492 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!69E3535B: (caller: 69D31A80) LogHr(1) tid(628) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!69E7B5DD: (caller: 69F112BF) Exception(1) tid(628) 8A15005E ] 
> 2024-04-07 23:38:06.492 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:13.899 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!55F8535B: (caller: 55E81A80) LogHr(1) tid(504) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!55FCB5DD: (caller: 560612BF) Exception(1) tid(504) 8A15005E ] 
> 2024-04-07 23:38:13.899 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:12.428 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!69DB535B: (caller: 69CB1A80) LogHr(1) tid(1410) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!69DFB5DD: (caller: 69E912BF) Exception(1) tid(1410) 8A15005E ] 
> 2024-04-07 23:38:12.429 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:19.626 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!55B0535B: (caller: 55A01A80) LogHr(1) tid(1060) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!55B4B5DD: (caller: 55BE12BF) Exception(1) tid(1060) 8A15005E ] 
> 2024-04-07 23:38:19.627 [REPO] Failed to open available source: msstore
> 2024-04-07 23:38:20.863 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x86 with version 14.38.33135.0

(Automated response - build 876.)

**Comment by stephengillie** [comment]
The authenticator window appears when running the full link, but the symlink appears to have no impact.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
@Exorcism0666 - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**